### ISSUE TITLE
Firmare app per Android

### ISSUE THREAD

**Issue by AlfioEmanueleFresta** [AlfioEmanueleFresta/marvin#4]
@ciopper90

**Comment by ciopper90** [comment]
Devo provare a firmare con la chiave un apk??

**Comment by AlfioEmanueleFresta** [comment]
@ciopper90 ti ho invitato per email a collaborare su Adobe Phonegap. 
https://build.phonegap.com/apps/1281266/builds
Vai sotto "Android", chiave, "Add key", e clicca "Rebuild".

**Comment by ciopper90** [comment]
siamo sicuri di voler assegnare la chiave per la firma direttamente alla piattaforma di build di adobe???(anche se non possiede la password possiede comunque la chiave)

**Comment by ciopper90** [comment]
ok..caricata ma non ho abbastanza diritti per collegarla con l'applicativo e quindi firmarlo. nella schemata non ho la possibilità di modificare nessuna informazione(mi devi avere aggiunto solo come tester) @AlfioEmanueleFresta

**Comment by AlfioEmanueleFresta** [comment]
Nope, ho appena controllato, sei sviluppatore. Inoltre non mi compare
alcuna chiave quindi non posso firmare neanche io!

Alfio

**Comment by AlfioEmanueleFresta** [comment]
(Le informazioni è normale siano readonly perché sono dal file di configurazione nel repository)

**Comment by ciopper90** [comment]
credo che i collaboratori non possano aggiungere la chiave.solo tu puoi aggiungere la chiave al repository per firmare

questo e come vedo la schermata
![marvin](https://cloud.githubusercontent.com/assets/5755844/6392326/ce2977dc-bdbf-11e4-89fe-12cbcda54318.png)
